test/unit/ui/popup.test.js

```diff
@@ -495,7 +495,7 @@ test('Popup#remove is idempotent (#2395)', (t) => {
     t.end();
 });
 
-test('Popup adds classes from className option', (t) => {
+test('Popup adds classes from className option, methods for class manipulations works properly', (t) => {
```

Comment:
Thanks for adding these tests!

Perhaps we should also have tests for edge cases, e.g. if someone passes an empty string (`''`) or a space-separated string (`'multiple class names'`) like they can pass to the `className` option? Would be great to capture the intended behavior for those cases.

Reply:
My idea for these methods was to keep it closer as possible to native `classList` methods. Native methods throws `DOMException` in these cases(`''` or `'multiple class names'`). And at this point we have 2 options:
1. Just add `t.throws` test for these cases
2. Add checks to my methods to prevent exceptions (e.g. `if (className) { this._container.classList.add(className) }`)

Which one is preferable?

Reply:
1 is fine - the idea is to use the tests to capture/document that we've thought about these edge cases & what behavior we expect, it's fine if we expect errors.

Reply:
tests added